lib/pdk/cli/config.rb

```diff
@@ -0,0 +1,20 @@
+module PDK::CLI
+  @config_cmd = @base_cmd.define_command do
+    name 'config'
+    usage _('config [subcommand] [options]')
+    summary _('Configure the Puppet Development Kit.')
+    default_subcommand 'get'
+
+    run do |_opts, args, _cmd|
+      if args == ['help']
+        PDK::CLI.run(%w[config --help])
+        exit 0
+      end
+
+      PDK::CLI.run(%w[config get]) if args.empty?
```

Comment:
Our usual behaviour is to have the default subcommand be `help` (with the exception of `pdk validate` which is a special case that combines the behaviour of all the subcommands). I would suggest that `help` be the default here as well for consistency.

Reply:
Fixed